packages/protocol/scripts/truffle/make-release.ts

```diff
@@ -1,17 +1,18 @@
 // tslint:disable: max-classes-per-file
 // tslint:disable: no-console
+// tslint:disable: ordered-imports
```

Comment:
is there any reason why not to order imports ?

Reply:
I think there's a mismatch between the commit hook and the lint that makes it fail